OpenRA.Mods.Common/Traits/Health.cs

```diff
@@ -19,9 +19,13 @@ public class HealthInfo : ITraitInfo, UsesInit<HealthInit>
 	{
 		[Desc("HitPoints")]
 		public readonly int HP = 0;
+
 		[Desc("Trigger interfaces such as AnnounceOnKill?")]
 		public readonly bool NotifyAppliedDamage = true;
 
+		[Desc("Distance above CenterPosition for targeting by weapons.")]
```

Comment:
I tend to agree that it may, but it also allows this feature to be used with actors whose graphics aren't perfectly vertical.

Imagine a Titan or similar with a weak spot that is offset from the centerpoint's Z axis.